RELEASE.md

```diff
@@ -0,0 +1,27 @@
+# Sourcegraph Helm Chart Release
+
+> Sourcegraph helm chart may be released outside the regular [Sourcegraph releases] schedule, these steps are required whenever releaseing a new version.
+
+## 1. Bump the `version` in `Chart.yaml`
+
+Doulbe check the `CHANGELOG.md` see if there is any `**BREAKING**:` changes and follow [semver] to bump chart `version` accordingly.
```

Comment:
```suggestion
Double check the `CHANGELOG.md` to see if there are any `**BREAKING**:` changes and follow [semver] to bump chart `version` accordingly. Increase the minor version for a breaking change or Sourcegraph upgrade, patch version for a backwards-compatible change.
```

Reply:
Sorry, that started out as a typo fix and then I figured "why not explain more". Feel free to ignore!